packages/admin-svelte/src/tokens.css

```
@@ -0,0 +1,537 @@
+  --adm-shadow-sm: 0 1px 3px rgba(11, 18, 32, 0.06), 0 1px 2px rgba(11, 18, 32, 0.04);
+  --adm-shadow-md: 0 6px 16px rgba(11, 18, 32, 0.08), 0 2px 6px rgba(11, 18, 32, 0.04);
+  --adm-shadow-lg: 0 24px 48px rgba(11, 18, 32, 0.14), 0 8px 16px rgba(11, 18, 32, 0.06);
+
+  /* Motion measured from the Ant Design reference build, then extended. */
+  --adm-ease-out: cubic-bezier(0.16, 1, 0.3, 1);
+  --adm-ease-std: cubic-bezier(0.645, 0.045, 0.355, 1);
+  --adm-dur-fast: 140ms;
+  --adm-dur: 200ms;
+  --adm-dur-slow: 300ms;
+
+  --adm-font: Inter, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)